docs/kyma-environment-broker/03-03-runtime-operations.md

```diff
@@ -42,7 +43,8 @@ The deprovisioning process contains the following steps:
 | Name                         | Domain         | Status      | Description                                                                            | Owner     |
 |------------------------------|----------------|-------------|----------------------------------------------------------------------------------------|-----------|
 | Deprovision_Initialization   | Deprovisioning | Done        | Initializes the `DeprovisioningOperation` instance with data fetched from the `ProvisioningOperation`. | @polskikiel (Team Gopher) |
-| Deprovision Azure Event Hubs | Event Hub      | In progress | Deletes the Azure Event Hub Namespace.                                                  | @montaro (Team SkyDivingTunas)   |
+| Deprovision Azure Event Hubs | Event Hub      | Done        | Deletes the Azure Event Hub Namespace.                                                  | @k15r (Team SkyDivingTunas)   |
+| Deprovision EMS              | EMS            | Done        | Unbinds and deprovisions the Enterprise Messaging instance using the Service Manager         | @k15r (Team SkyDivingTunas)     |
```

Comment:
```suggestion
| Deprovision EMS              | EMS            | Done        | Unbinds and deprovisions the Enterprise Messaging instance using the Service Manager.         | @k15r (Team SkyDivingTunas)     |
```
Very very small change, but all the other entries end with a period so it would be nice if it's consistent.